release/release_gold.sh

```diff
@@ -132,6 +132,28 @@ function tag_release {
 
 		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
 	fi
+
+	if (! curl \
+			"https://api.github.com/repos/liferay/${repository}/git/refs" \
+			--data-raw '
+				{
+					"message": "",
```

Comment:
@lucasmiranda0, can you please create a new method with 2 parameters (url and data-raw) and use it to invoke the 2 curls?